src/prover/index.ts

```diff
@@ -263,7 +292,7 @@ export class Prover {
     return new Proof(this.blueprint, proofProps);
   }
 
-  private async _incNumLocalProofs(): Promise<void> {
+  async _incNumLocalProofs(): Promise<void> {
```

Comment:
if we are removing it from private, it would be better to remove the `_` prefix as well